### PR TITLE
add pleo.io

### PR DESCRIPTION
**Pull request by christian-steinmeyer**
Hi! I wasn't too sure about the supported methods, so I only added the two that I was sure about, please have a look and add additional ones that I missed.

### CODE REVIEW

**Comment by phallobst** [comment]
We prefer to use icons in  SVG format whenever possible:
pleo io

File: entries/p/pleo.io.json
Comment (phallobst) [inline entries/p/pleo.io.json]:
```suggestion
  "Pleo": {
```

File: entries/p/pleo.io.json
Comment (phallobst) [inline entries/p/pleo.io.json]:
```suggestion
    "documentation": "https://help.pleo.io/en/articles/5628169",
```

**Review by phallobst — CHANGES_REQUESTED** [review]

**Comment by Carlgo11** [comment]
Thanks for the pull request @christian-steinmeyer,
The documentation page you linked to states:
>When you log into Pleo, you'll need to enter: 
    1. Your email address
    2. A 4-digit passcode
    3. A 6-digit authentication code

Is the 4-digit passcode something that's sent by email? Also, are passwords not required?

**Comment by christian-steinmeyer** [comment]
Thanks for the comments!
The four digit passcode is set during the registration process. As I understand it, this passcode is used like a password. No idea why. I'm only my mobile device right now. If you add a commit to my branch (maintainers should be able to) with the image change, feel free to do so, otherwise I'll do it tomorrow. And please squash before merging :D

**Review by phallobst — APPROVED** [review]

**Pull request merged**
